package.json

```diff
@@ -14,14 +15,17 @@
   "author": "",
   "license": "ISC",
   "peerDependencies": {
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
   },
   "devDependencies": {
     "@babel/preset-react": "^7.0.0",
+    "@types/react": "^17.0.38",
+    "@types/react-dom": "^17.0.11",
```

Comment:
version upgrade cannot be done on this PR, because then users using version 16.x will have an error while installation (however, as it happens now on versions 17 and 18).
I think the compatibility issue should be solved in separate PR